### DISCUSSION TITLE
Hello, I'm Zosh (the new designer)

### DISCUSSION THREAD

**Discussion by zcolah** [zesty-io/manager-ui#1129] [General]
Hello everyone, 
I'm Zosh, the new designer here at Zesty. I will be working closely with the product team to make the platform a joyful easy to use experience for all. I will be active here and am keen to learn all the ways we can make Zesty a great experience for everyone. I will be starting tomorrow and am super excited to meet everyone 🥳. 

**A little bit about me:** 
-  I design interfaces, do research, manage products, develop websites, analyze data, teach courses, create ads, and am even a founder
- I was most recently at Pathrise, where I led the design of our learning management system for 2 years, designing solutions to alleviate the pain people feel in the job search 
- In my spare time, you'll find me teaching or iterating on [UXLibrary.org](https://www.uxlibrary.org/) , a website for people to discover UX resources

I am always happy to help and available on zoshua@zesty.io.

**Comment by ardeay** [comment]
Heya @zcolah we are very excited to have you on board!
